hicommonwealth/commonwealth
# User-Onboarding: Added flow step for existing wallet onboarding

## Link to Issue
Closes: 
- https://github.com/hicommonwealth/commonwealth/issues/6643
- https://github.com/hicommonwealth/commonwealth/issues/6644

## Description of Changes
- Adds user flow step for existing wallet and sso onboarding

## "How We Fixed It"
N/A

## Test Plan

- Verify that existing login and create account flows work.
- Enable `FLAG_USER_ONBOARDING_ENABLED` in `.env`.
- Add `type='create-account'` to `<AuthModal/>` instance in `SubLayoutHeader.tsx`, apply this diff
```diff
diff --git a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
index 4564d26340..c58647b700 100644
--- a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
+++ b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
@@ -59,6 +59,7 @@ export const SublayoutHeader = ({
         open={isFeedbackModalOpen}
       />
       <AuthModal
+        type='create-account'
         onClose={() => setIsAuthModalOpen(false)}
         isOpen={isAuthModalOpen}
       />

```
and open the modal
  - click on "I have a wallet" and verify it matches [figma](https://www.figma.com/file/I5jbkBPzNrbYMHYQJb6CNY/Onboarding?type=design&node-id=802-5349&mode=design&t=LF7SKB8busRGTSRc-4)
  - click on "Create a wallet for me" and verify it matches [figma](https://www.figma.com/file/I5jbkBPzNrbYMHYQJb6CNY/Onboarding?type=design&node-id=802-5904&mode=design&t=wgb1cglV90zGNK1C-4)

- Login with both wallets and sso options and verify login works for both new and existing accounts

## Deployment Plan
N/A

## Other Considerations
N/A
